assignment/scripts/aboutMe.js

```diff
@@ -1,59 +1,91 @@
 // REQUIRED FEATURES:
 // 1 - Create a variable called `firstName` and assign it the value of your first name
-
+// used const for name and number since those do not change;
+const firstName = "Michael";
 // 2 - Create a second variable called `lastName` and assign it the value of your last name
-
+const lastName = "Duren";
 // 3 - Create a third variable called `fullName`, assign it the value of your first and last name
 // (remember, you already have variables for this, can you use those?)
-
+const fullName = `${firstName} ${lastName}`;
 // 4 - Console log the value of `fullName`
-
+console.log(fullName);
 // 5 - Create a variable called `luckyNumber` and assign it the value of your lucky number.
-
-// 6 - Console log this sentence, adding in the variables you created above: 
+const luckyNumber = 8;
+// 6 - Console log this sentence, adding in the variables you created above:
 // 'My name is (full name), and I think (lucky number) is a winner!'.
 // Refer back to the videos if you need help with this one.
-
+console.log(`My name is ${fullName}, and I think ${luckyNumber} is a winner!`);
 // 7 - Create a variable named `adventurous` and set it to a boolean value (true or false)
-
+let adventures = true;
 // 8 - Create a variable named `food`, and set its value to a string of your favorite food
-
+let food = "Burrito";
 // 9 - Create a variable called `pets` and set it to the value of the number of pets you have
-
+let pets = 1;
 // 10 - Create a variable called `friendsPets` and assign it the value of the number of pets your friend has
-
+let friendsPets = 2;
 // 11 - Add two pets to your `pets` variable
-
+let allPets = friendsPets + pets;
 // 12 - Create a constant variable called `allowedPets` and set it to a number value of your choice
-
-// 13 - Create a conditional: if adventurous is true, console log "Adventures are great!", 
+const allowedPets = 3;
+// 13 - Create a conditional: if adventurous is true, console log "Adventures are great!",
 // if it's not true, console log "How about we stay home?"
-
-
+adventures ? console.log("Adventures are great!") : console.log("How about we stay at home ? ");
```

Comment:
Okay, so the key difference between a ternary and an if-block is that a ternary is _short_, but that a ternary is an _expression_. In other words, it outputs a value. Although this code works, a better approach would be to use that output, such as by saving it to a variable.

```js
let message = adventures ? 'Adventures are great!' : 'How about we stay home?';
console.log(message);
```

Or even putting the output directly into the console.log:

```js
console.log(adventures ? 'Adventures are great!' : 'How about we stay home?');
```